common/summary/summary-modifier.py

```diff
@@ -45,7 +45,8 @@
              "OIDC Password Grant Type Retrieve User Attributes Groups and Roles": "1 GetToken_Password_Grant",
              "SAML2 SSO Redirect Binding": "2 Identity Provider Login",
              "Token Exchange Grant": "1 GetToken_Token_Exchange_Grant",
-             "B2B OIDC Auth Code Grant Redirect With Consent": "4. Common Auth Login Request to Sub Org"}
+             "B2B OIDC Auth Code Grant Redirect With Consent": "4. Common Auth Login Request to Sub Org",
```

Comment:
Why we have a dot here only for this line?

Reply:
This is related to a different test script (use case) that was added. I noticed that the name of the 4th request in the test script includes a dot.

Reply:
ack